R/drop_upload.R

```diff
@@ -23,37 +27,59 @@
 #' drop_upload("mtt.csv")
 #'}
 drop_upload <- function(file,
-                        dest = NULL,
-                        overwrite = TRUE,
-                        autorename = FALSE,
+                        path = NULL,
+                        mode = "overwrite",
+                        autorename = TRUE,
                         verbose = FALSE,
-                        encode = "multipart",
+                        mute = FALSE,
```

Comment:
if we keep `verbose`, the API-specific args should all be before it

Reply:
do we plan to support `client_modified`? worry about it later?

Reply:
Will fix `verbose`. Yes I skipped `client_modified` for now since there was no clear use case for for this round.

Reply:
Fixed up.